src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -114,23 +144,166 @@ const ReservationList: FC<ReservationListProps> = ({ pageSize }) => {
     !isLoadingFBS &&
     !isLoadingPublizon;
 
+  const openReservationDeleteModal = (deleteId: string | number | null) => {
```

Comment:
Why number.  Modal id's are always strings I guess

Reply:
same as above, changed to string